test/luatest_helpers/server.lua

```diff
@@ -78,7 +81,8 @@ end
 -- @return map
 function Server:build_env()
     local res = getmetatable(getmetatable(self)).build_env(self)
-    if self.box_cfg ~= nil then
+    -- Best way to check if table is not empty in lua I guess...
+    if not rawequal(next(self.box_cfg), nil) then
```

Comment:
It can be simpler: `if not next(self.box_cfg) then ...`. Why do you need `rawequal()`?

Reply:
Just in case someone overloaded __eq method for self.box_cfg.